pkg/ebpf/verifier/calculator/main.go

```diff
@@ -53,6 +70,18 @@ func main() {
 			return nil
 		}
 
+		if len(filterFiles) > 0 {
+			found := false
+			for _, f := range filterFiles {
+				if strings.TrimSuffix(d.Name(), ".o") == f {
+					found = true
+				}
+			}
+			if !found {
```

Comment:
This only guarantees that one of the filter files is valid, not that all of them are.